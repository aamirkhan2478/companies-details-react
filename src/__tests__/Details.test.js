import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../Redux/configureStore';
import DetailPage from '../Pages/DetailsPage';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <DetailPage />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
