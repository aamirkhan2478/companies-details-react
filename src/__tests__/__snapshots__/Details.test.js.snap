// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="main-card"
  >
    <div>
      <img
        alt="company-logo"
        height={100}
        width={150}
      />
    </div>
    <div>
      <h2 />
      <span>
        (
        )
      </span>
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Symbol
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        CEO
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Country
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        City
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Price
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Range
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Visit Website
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4>
        <a
          target="_black"
        >
          Go to Website
        </a>
      </h4>
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Address
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Full Time Employees
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Industry
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
  <div
    className="row-1"
  >
    <div
      className="col-sm-6"
    >
      <h4>
        Phone
      </h4>
    </div>
    <div
      className="col-sm-6"
    >
      <h4 />
    </div>
  </div>,
]
`;
